pages/auth/login.tsx

```diff
@@ -12,10 +12,13 @@ import { Confirmation } from "@components/auth/Confirmation/Confirmation";
 import UserNavLayout from "@components/globals/layouts/UserNavLayout";
 import * as Yup from "yup";
 
-const Register = () => {
-  const { username, cognitoError, setCognitoError, login } = useAuth();
+const Login = () => {
+  const { cognitoError, setCognitoError, login } = useAuth();
   const { t } = useTranslation(["login", "common"]);
```

Comment:
Looks like strings "UsernameOrPasswordIncorrect" is in congnito-errors, may need the namespace below

Reply:
But not worth stopping a PR over IMO

Reply:
Nice catch! It was still working when I tested the branch though

Reply:
Thanks for the catch.  Adding in the missing namespace.